### PR TITLE
update dashboard community deployment

### PR DESCRIPTION
**Pull request by cooper-lzy**
### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: docs-2.0/nebula-dashboard/2.deploy-dashboard.md [hunk 1]
```diff
@@ -20,329 +22,125 @@ Before you deploy Dashboard, you must confirm that:
...
+  |prometheus |9090| The time series database that stores monitoring data. |
+
+  The descriptions of the configuration file are as follows.
+
+  ```yaml
+  port: 7003   # Web service port.
+  gateway:
+    ip: hostIP   # The IP of the machine where the Dashboard is deployed.
+    port: 8090
+    https: false  # Whether the port is the type of HTTPS.
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard/2.deploy-dashboard.md:53]:
```suggestion
    https: false  # Whether to enable HTTPS.
```

File: docs-2.0/nebula-dashboard/2.deploy-dashboard.md [hunk 2]
```diff
@@ -20,329 +22,125 @@ Before you deploy Dashboard, you must confirm that:
...
+    runmode: dev  # Program running mode, including dev, test, and prod. It is used to distinguish between different running environments generally.
+  stats-exporter:
+    ip: hostIP   # The IP of the machine where the Dashboard is deployed.
+    nebulaPort: 9200
+    https: false  # Whether the port is the type of HTTPS.
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard/2.deploy-dashboard.md:58]: ditto

File: docs-2.0/nebula-dashboard/2.deploy-dashboard.md [hunk 3]
```diff
@@ -20,329 +22,125 @@ Before you deploy Dashboard, you must confirm that:
...
+  node-exporter:
+    - ip: nebulaHostIP_1 # The IP of the machine where the NebulaGraph is deployed.
+      port: 9100
+      https: false # Whether the port is the type of HTTPS.
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard/2.deploy-dashboard.md:62]: ditto

File: docs-2.0/nebula-dashboard/2.deploy-dashboard.md [hunk 4]
```diff
@@ -20,329 +22,125 @@ Before you deploy Dashboard, you must confirm that:
...
+| Parameter                  | Description       |
+| :------------------------- | :------------------- |
+| `dashboard_path` | Dashboard installation path.  |
+| `-v`                       | Display detailed debugging information.   |
+| `-h`                       | Display help information.        |
+| `start`                    | Start the target services.       |
+| `restart`                  | Restart the target services.       |
+| `stop`                     | Stop the target services.           |
+| `status`                   | Check the status of the target services.       |
+| `prometheus`               | Set the prometheus Service as the target service. |
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard/2.deploy-dashboard.md:135]:
```suggestion
| `prometheus`               | Set the prometheus service as the target service. |
```

File: docs-2.0/nebula-dashboard/2.deploy-dashboard.md [hunk 5]
```diff
@@ -20,329 +22,125 @@ Before you deploy Dashboard, you must confirm that:
...
+| `webserver`                | Set the webserver Service as the target service.  |
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard/2.deploy-dashboard.md:136]: change all the following Service to service plz.

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/cluster-information/job-management.md [hunk 6]
```diff
@@ -6,6 +6,11 @@ Users can manage the jobs in a specified graph space through the Dashboard, incl
 
     How to run jobs, see [Job manager and the JOB statements](../../../3.ngql-guide/4.job-statements.md).
 
+## Prerequisites
+
+- NebulaGraph Enterprise Edition 3.4.0 and above supports job management.
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/cluster-information/job-management.md:11]:
```suggestion
- The job management feature is only available in NebulaGraph Enterprise Edition 3.4.0 and above versions
```

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/cluster-information/job-management.md [hunk 7]
```diff
@@ -6,6 +6,11 @@ Users can manage the jobs in a specified graph space through the Dashboard, incl
...
+- NebulaGraph Community Edition 3.3.0 and above supports job management.
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/cluster-information/job-management.md:12]: ditto

File: docs-2.0/nebula-dashboard/2.deploy-dashboard.md [hunk 8]
```diff
@@ -20,329 +22,125 @@ Before you deploy Dashboard, you must confirm that:
...
+
+## Steps
+
+1. Download the tar package[nebula-dashboard-{{ dashboard.release }}.x86_64.tar.gz](https://oss-cdn.nebula-graph.com.cn/nebula-graph-dashboard/{{ dashboard.release }}/nebula-dashboard-{{ dashboard.release }}.x86_64.tar.gz)  as needed.
+
+2. Run `tar -xvf nebula-dashboard-{{ dashboard.release }}.x86_64.tar.gz` to decompress the installation package.
+
+3. Modify the `config.yaml` file in `nebula-dashboard`.
+
+  The configuration file contains the configurations of four dependent services and clusters. The descriptions of the dependent services are as follows.
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard/2.deploy-dashboard.md:37]:
```suggestion
  The configuration file contains the configurations of four dependent services and configurations of clusters. The descriptions of the dependent services are as follows.
```

**Review by abby-cyber — APPROVED** [review]

**Review by foesa-yang — APPROVED** [review]

**Pull request merged**